magento/pwa-studio
# [PWA-239] Shipping Method (Cart)

## Description

When a user (shopper) is in the cart flow for Venia, a shipping method selection/estimate component should display in order to allow a shopper to determine the impact of shipping to their order summary. 

 
Requirements: 

Adhere to design specs/flows
Initial design accounts for unauthenticated cart/checkout, authenticated to be a fast follow
out of scope: 

detecting location based on customer device

## Related Issue





* [[PWA-239](https://jira.corp.magento.com/browse/PWA-239)] Shipping Method (Cart)

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


If you want to verify that third-party shipping providers are working, you'll need inventory items that have weight, and your shipping methods configured in the backend. I've done this with a staging environment, you'll have to use Carmina Necklace: https://mcstaging-jnz3dtiuj77ca.dummycachetest.com/
1. Add product to your cart
2. Navigate to `/cart` page and verify that estimate shipping fields are displayed (country, state, zip)
3. Enter values for these fields and shipping methods should be displayed based on available methods
4. Select a method and verify that PriceSummary is updated to reflect this selection

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
